esi_report.sh

```diff
@@ -27,18 +27,18 @@ elif [[ $1 == "leases-monthly" ]]; then
 
 elif [[ $1 == "nodes-daily" ]]; then
     date=$(date +"%Y-%m-%d")
-    output_file=${report_dir}/nodes/daily/${date}_daily.log
+    output_file=${report_dir}/nodes/daily/${date}_daily.csv
     echo "===== Start generating report at $(date +"%Y-%m-%dT%H:%M:%S") =====" >> $output_file
```

Comment:
Can you move these echo statements into their own log file, and also have them output the command being used? We don't want these statements *in* the CSV itself, as that corrupts the CSV file; and it'd be nice to be able to rerun commands based on the log file.

Reply:
Okay, will do!